### ISSUE TITLE
Softmax in word language example

### ISSUE THREAD

**Issue by cedricdeboom** [pytorch/examples#254]
I was wondering why there isn't a softmax nonlinearity applied at the final output layer of the word language example:
https://github.com/pytorch/examples/blob/62d5ca57af2c33c96c40010c115e5ff34136abb5/word_language_model/model.py#L20
Doesn't this skew the cross entropy loss function?

**Comment by soumith** [comment]
The CrossEntropy loss function has a Log + SoftMax + NegativeLogLikehood
